app/api/models/errors/DownstreamErrors.scala

```diff
@@ -36,8 +37,4 @@ object DownstreamErrors {
 
 case class OutboundError(error: MtdError, errors: Option[Seq[MtdError]] = None) extends DownstreamError
 
-object DownstreamError
-    extends MtdError(
-      code = "INTERNAL_SERVER_ERROR",
-      message = "An internal server error occurred"
-    )
+object DownstreamError extends MtdError("INTERNAL_SERVER_ERROR", "An internal server error occurred", INTERNAL_SERVER_ERROR)
```

Comment:
This isn't really part of the PR, also it seems to be done this same way in many of the APIs (at least the ones I've checked so far). I agree it could be improved, probably the wrong time for that particular refactoring though, as it would affect quite a lot of code.

Reply:
As a separate thing I'm renaming StandardDownstreamError to InternalError though (although I still think it's the wrong name!)